### PR TITLE
Added new parameter to Swiper wrapper to limit breakpoint intervention fixes #10525

### LINKED ISSUE #10525: Elementor seems to break swiperJS (5.3.0) breakpoint settings (swiper navigation not working)
## Prerequisites
- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description
Elementor seems to mess with the swiperJS breakpoint settings. While all work fine when no breakpoint settings are set, the swiper is broken once the breakpoint settings are used. Elementor currently uses swiper 5.3.0 - i tried with Swiper 5.3.1 and 5.2.0 with the same results.

A very strange bug. Did Elementor use monkey patching to override some swiper code on the fly?

## Steps to reproduce
- Add an HTML-Widget on any elementor page.
- Paste the following code
- Check / Test
  - navigation (left/right arrows) are visible but not clickable
  - pagination bubbles are not in the correct number
- You may switch the breakpoint settings off by add a opening multiline-comment ("/*") in line 35. Then the swiper will work as intended - but will have no breakpoint control.
```
<div class="swiper-container test-swiper-container" style="width: 100%; height: 300px;">
    <div class="swiper-wrapper">
        <div class="swiper-slide" style="background:#f00">Slide 1</div>
        <div class="swiper-slide" style="background:#0f0">Slide 2</div>
        <div class="swiper-slide" style="background:#00f">Slide 3</div>
        <div class="swiper-slide" style="background:#a00">Slide 4</div>
        <div class="swiper-slide" style="background:#900">Slide 5</div>
        <div class="swiper-slide" style="background:#300">Slide 6</div>
        <div class="swiper-slide" style="background:#000">Slide 7</div>
        <div class="swiper-slide" style="background:#fff">Slide 8</div>
        <div class="swiper-slide" style="background:#f3f">Slide 9</div>
        <div class="swiper-slide" style="background:#f9f">Slide 10</div>
    </div>
    <!-- Add Pagination -->
    <div class="swiper-pagination"></div>
    <!-- If we need navigation buttons -->
    <div class="swiper-button-prev"></div>
    <div class="swiper-button-next"></div>
</div>

<script>

jQuery(document).ready(function($) {
    var swiper = new Swiper('.swiper-container', {
        slidesPerView: 1,
        spaceBetween: 10,
        // init: false,
        pagination: {
            el: '.swiper-pagination',
            clickable: true,
        },
        navigation: {
            nextEl: '.swiper-container .swiper-button-next',
            prevEl: '.swiper-container .swiper-button-prev',
        },
        
        breakpoints: {
            640: {
                slidesPerView: 2,
                spaceBetween: 20,
            },
            768: {
                slidesPerView: 4,
                spaceBetween: 40,
            },
            1024: {
                slidesPerView: 5,
                spaceBetween: 50,
            },
        }//*/
    });
});
</script>
```

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment
Wordpress 5.3.2
Elementor 2.9.0

### PR DESCRIPTION
**Pull request by udidol**
## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Added new parameter `handleElementorBreakpoints` to Elementor's Swiper wrapper to limit its breakpoint intervention

## Quality assurance

- [X] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

**Comment by shilo-ey** [comment]
fixes #10525

**Comment by mackyangeles** [comment]
How’s the bug? Is it near fixing?

**Comment by udidol** [comment]
The issue will be addressed in an upcoming release

**Comment by mackyangeles** [comment]
> The issue will be addressed in an upcoming release

Still not fix on version 2.9.3 (just did the update) :(

**Comment by udidol** [comment]
> > The issue will be addressed in an upcoming release
> 
> Still not fix on version 2.9.3 (just did the update) :(

Upcoming, but not this one. We will update the ticket once the change is implemented.

**Comment by vollyimnetz** [comment]
Hi @KingYes, hi @kobizz,
it would be really great to get this change into elementor **fast**. 
Every plugin- or theme-developer who used the swiper library for a custom functionality, tearing there hair because elementor destroyed the breakpoint settings.

Please merge the request to keep the code clean from custom overrides.

The problems are also described here: https://github.com/elementor/elementor/issues/10525

File: assets/lib/swiper/swiper.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
 /**
- * Swiper 5.3.0
+ * Swiper 5.3.6
```
Comment (kobizz) [inline assets/lib/swiper/swiper.js]: Update version inside related files

**Review by kobizz — CHANGES_REQUESTED** [review]

File: assets/lib/swiper/swiper.js (hunk 1 above)
Comment (udidol) [reply assets/lib/swiper/swiper.js]: Version numbers are already updated...

**Review by kobizz — APPROVED** [review]

**Comment by mackyangeles** [comment]
Any update on this? With 2.9.4 not yet fix... we need a fix ASAP

**Review by KingYes — APPROVED** [review]

**Comment by shilo-ey** [comment]
We released Pro v2.8.5 to support this fix and will soon release a patch to this issue.

**Pull request merged**